pydantic/fields.py

```diff
@@ -850,3 +850,7 @@ def PrivateAttr(
         default,
         default_factory=default_factory,
     )
+
+
+class DeferredType:
+    """Used to postpone field preparation, while creating recursive generic models."""
```

Comment:
```suggestion
    """
    Used to postpone field preparation, while creating recursive generic models.
    """
```

Reply:
I'm not sure this type should live here, can you move it into `generics.py`?

Reply:
This would introduce a circular import, since `ModelField` operates on `DeferredType`, and `generics.py` is using `ModelField`.

Are you sure `fields.py` isn't a right place for it?
For me it seems like it's a feature of `ModelField`.

Reply:
You could use deferred imports, but ok.

Please can you fix the docstring as per the suggestion.

Reply:
I've fixed all docstrings in the PR